src/hooks/useAssets.ts

```diff
@@ -21,14 +21,14 @@ function assetsLoadedTest<T>(asset: UnresolvedAsset<T>)
 /** Loads assets, returning a hash of assets once they're loaded. */
 export function useAssets<T = any>(
     /** @description Assets to be loaded. */
-    assets: UnresolvedAsset<T>[],
+    assets: UnresolvedAsset[],
```

Comment:
In the Assets.load implementation they do not type UnresolvedAsset, this is what was causing the type to be inferred from the data.

https://github.com/pixijs/pixijs/blob/b74ec8214da0ca90d870adb7b71efd876a86b1f1/src/assets/Assets.ts#L450-L461